sub-packages/bionemo-amplify/src/bionemo/amplify/train_amplify.py

```diff
@@ -236,7 +236,7 @@ def main(
 
     # Initialize the data module.
     data = AMPLIFYDataModule(
-        train_hf_dataset=hf_load_dataset("chandar-lab/UR100P", data_dir="UniProt", split="train"),  # type: ignore
+        train_hf_dataset=hf_load_dataset("chandar-lab/UR100P", split="train"),  # type: ignore
```

Comment:
how the data is handled, where it is downloaded? or is it provided by hf on-the-fly?

Reply:
Yeah it downloads to your huggingface cache. I added a comment to that effect.